sql/size_utils.sql

```diff
@@ -428,6 +429,19 @@ DECLARE
     max_compressed_row_count BIGINT = 1000;
     is_compressed_chunk INTEGER;
 BEGIN
+    -- Check if input relation is continuous aggregate view then
+    -- get the corresponding materialized hypertable and schema name
+    SELECT format('%I.%I', ht.schema_name, ht.table_name)::regclass FROM pg_class c
+    INTO mat_ht
+    INNER JOIN pg_namespace n ON (n.OID = c.relnamespace)
+    INNER JOIN _timescaledb_catalog.continuous_agg a ON (a.user_view_schema = n.nspname AND a.user_view_name = c.relname)
+    INNER JOIN _timescaledb_catalog.hypertable ht ON (a.mat_hypertable_id = ht.id)
+    WHERE c.OID = relation;
```

Comment:
```suggestion
    SELECT format('%I.%I', ht.schema_name, ht.table_name)::regclass
    INTO mat_ht
    FROM pg_class c
    JOIN pg_namespace n ON (n.OID = c.relnamespace)
    JOIN _timescaledb_catalog.continuous_agg a ON (a.user_view_schema = n.nspname AND a.user_view_name = c.relname)
    JOIN _timescaledb_catalog.hypertable ht ON (a.mat_hypertable_id = ht.id)
    WHERE c.OID = relation;
```

Nit. just cosmetic SQL formatting.

It is interesting that your SQL lead to a syntax error on psql console:
```sql
26528 (leader) fabrizio=# SELECT format('%I.%I', ht.schema_name, ht.table_name)::regclass FROM pg_class c INTO mat_ht
    INNER JOIN pg_namespace n ON (n.OID = c.relnamespace)
    INNER JOIN _timescaledb_catalog.continuous_agg a ON (a.user_view_schema = n.nspname AND a.user_view_name = c.relname)
    INNER JOIN _timescaledb_catalog.hypertable ht ON (a.mat_hypertable_id = ht.id)
    WHERE c.OID = 'conditions_summary'::regclass;
ERROR:  42601: syntax error at or near "INTO"
LINE 1: ...ma_name, ht.table_name)::regclass FROM pg_class c INTO mat_h...
                                                             ^
LOCATION:  scanner_yyerror, scan.l:1188
```

Probably it is a `psql` issue because your tests are passing, but with proposed reformatting it works perfect:
```sql
26528 (leader) fabrizio=# SELECT format('%I.%I', ht.schema_name, ht.table_name)::regclass
    INTO mat_ht
    FROM pg_class c
    JOIN pg_namespace n ON (n.OID = c.relnamespace)
    JOIN _timescaledb_catalog.continuous_agg a ON (a.user_view_schema = n.nspname AND a.user_view_name = c.relname)
    JOIN _timescaledb_catalog.hypertable ht ON (a.mat_hypertable_id = ht.id)
    WHERE c.OID = 'conditions_summary'::regclass;
SELECT 1
```

Reply:
Thanks @fabriziomello, I have formatted the SQL as suggested.